src/_locales/pl/messages.yml

```diff
@@ -440,7 +442,7 @@ menuFindScripts:
   message: Znajdź skrypty dla tej strony
 menuMatchedFrameScripts:
   description: Label for menu listing matching scripts in sub-frames.
-  message: ''
+  message: Skrypty tylko w ramkach pomocniczych 
```

Comment:
Is the trailing space kept intentionally? If so, quotes are needed.

Reply:
Mistake. I corrected it. Excuse me.

PS. When will I see newly added translations in Violentmonkey?

Reply:
In the next release, I guess in a few days.

Reply:
OK, thanks for info.